web-ng/src/app/components/feature-panel/feature-panel.component.scss

```diff
@@ -14,31 +14,39 @@
  * limitations under the License.
  */
 
+@import '_variables';
+
 .observation.card {
-    margin: 16px;
+  margin: 16px;
 }
 .observation .field-label {
-    font-size: 12px;
-    line-height: 16px;
-    color: #70757A;
-    margin-bottom: 3px;
+  font-size: 12px;
+  line-height: 16px;
+  color: #70757a;
```

Comment:
Please use Angular themes to retrieve colors defined in https://github.com/google/ground-platform/blob/master/web-ng/src/app/shared/styles/_variables.scss. 

Examples:
https://github.com/google/ground-platform/search?q=mat-color